src/docfx/build/xref/InternalXrefSpec.cs

```diff
@@ -21,12 +22,17 @@ internal class InternalXrefSpec : IXrefSpec
 
         string IXrefSpec.Uid => Uid.Value;
 
-        public InternalXrefSpec(SourceInfo<string> uid, string href, Document declaringFile, MonikerList monikerList)
+        // TODO: change to use xrefSpec type to express what kind of xref spec it is: e.g. achievement, module
+        [JsonIgnore]
+        internal string? DeclaringPropertyName { get; }
```

Comment:
Technically this is the full JSON token path

```suggestion
        internal string? DeclaringPropertyPath { get; }
```